dev_tools/notebooks/isolated_notebook_test.py

```diff
@@ -177,7 +177,7 @@ def test_notebooks_against_released_cirq(partition, notebook_path, cloned_env):
 
     notebook_file = os.path.basename(notebook_path)
 
-    rewritten_notebook_descriptor, rewritten_notebook_path = rewrite_notebook(notebook_path)
+    rewritten_notebook_path = rewrite_notebook(notebook_path)
```

Comment:
It seems we are (and have always been) leaking temporary files. Could you fix while you're at it?

Reply:
done.

Reply:
@viathor - thank you for the review.  The temporary files cleanup was a bit more involved - can you please take a quick look again?